### ISSUE TITLE
No result if formatting has opening brace against interface name eg.  'interface <interface-name>{' 

### ISSUE THREAD

**Issue by jrkd** [Box-Of-Hats/ts-csharp#6]
This works:
```
interface MyTypescriptInterface {
    propOne: any;
    propTwo: string;
    propThree: number[];
    propFour: boolean;
}

class AnActualClass {
    aProperty: string;
}

interface AnotherTypescriptInterface {
    nestedObjectsInAList: MyTypescriptInterface[];
    recursiveObject: AnotherTypescriptInterface;
    isReallyCool: boolean;
}
```

But removing the space between interface name and opening brace does not, but is valid typescript:
```
interface MyTypescriptInterface{
    propOne: any;
    propTwo: string;
    propThree: number[];
    propFour: boolean;
}

class AnActualClass{
    aProperty: string;
}

interface AnotherTypescriptInterface{
    nestedObjectsInAList: MyTypescriptInterface[];
    recursiveObject: AnotherTypescriptInterface;
    isReallyCool: boolean;
}
```

**Comment by jrkd** [comment]
Clone of https://github.com/Box-Of-Hats/vscode-ts-to-cs/issues/9 after I realized the parsing code is in ts-charp repo

**Comment by Box-Of-Hats** [comment]
Merged PR and published version `0.3.1` to [npm](https://www.npmjs.com/package/ts-csharp).
